sbin/build.sh

```diff
@@ -56,50 +55,53 @@ function parseArguments() {
 }
 
 # Add an argument to the configure call
-addConfigureArg()
-{
+addConfigureArg() {
   # Only add an arg if it is not overridden by a user-specified arg.
   if [[ ${BUILD_CONFIG[CONFIGURE_ARGS_FOR_ANY_PLATFORM]} != *"$1"* ]] && [[ ${BUILD_CONFIG[USER_SUPPLIED_CONFIGURE_ARGS]} != *"$1"* ]]; then
     CONFIGURE_ARGS="${CONFIGURE_ARGS} ${1}${2}"
   fi
 }
 
 # Add an argument to the configure call (if it's not empty)
-addConfigureArgIfValueIsNotEmpty()
-{
+addConfigureArgIfValueIsNotEmpty() {
   # Only try to add an arg if the second argument is not empty.
   if [ ! -z "$2" ]; then
     addConfigureArg "$1" "$2"
   fi
 }
 
 # Configure the boot JDK
-configuringBootJDKConfigureParameter()
-{
+configureBootJDKConfigureParameter() {
   addConfigureArgIfValueIsNotEmpty "--with-boot-jdk=" "${BUILD_CONFIG[JDK_BOOT_DIR]}"
 }
 
+# Configure the Shenandoah GC build parameter for JDK 11-15
+configureShenandoahBuildParameter() {
+  if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ]; then
+    if [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" -ge "${JDK11_CORE_VERSION}" ] && [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" -le "${JDK15_CORE_VERSION}" ]; then
+      addConfigureArgIfValueIsNotEmpty "--with-jvm-features=" "shenandoahgc"
```

Comment:
Two comments:
- Why use the `IfValueIsNotEmpty` verison of `addConfigureArg` when it's explicitly not-empty?
```suggestion
      addConfigureArg "--with-jvm-features=" "shenandoahgc"
```
Also the nested if as opposed to just `&&`'ing them together on the same line seems to me to make it less readable, unless the intention is to adjust it to change it to an "or" of other variants going forward (but if that's the intention I would suggest splitting it out at that time)